src/nsrt_learning/nsrt_learning_main.py

```diff
@@ -106,7 +106,7 @@ def _learn_pnad_options(pnads: List[PartialNSRTAndDatastore],
     # same known parameterized option, or all actions should have no option.
     known_option_pnads, unknown_option_pnads = [], []
     for pnad in pnads:
-        if not pnad.datastore:
+        if not pnad.datastore:  # pragma: no cover
```

Comment:
i'm confused by this... how can this arise in light of the change you made to `nsrt_learning_main.py`? can we change this to an assert instead?